docs/Installation.md

```diff
@@ -419,9 +420,17 @@ service zm_testagent status
 See the [post-installation] section for post-installation matters.
 
 
-## 6. Post-installation
+## 6. Installation on CentOS 7

```

Comment:
The official name is CentOS Linux, which will be used in the main Zonemaster README (https://github.com/zonemaster/zonemaster/pull/1060)

Reply:
fixed